test/e2e/framework/skipper/skipper.go

```diff
@@ -46,6 +46,9 @@ func Skipf(format string, args ...interface{}) {
 	panic("unreachable")
 }
 
+// Skip is an alias for ginkgo.Skip.
+var Skip = ginkgo.Skip
```

Comment:
We should probably make this behave more like Skipf, where we pass the caller info?

Reply:
What do you mean with "pass the caller info"?

Both `Skipf` and this `Skip` record their direct caller as location where the test gets skipped.

Reply:
@BenTheElder Do you mean to make sure the call stack is skipped by one to make sure the location of caller is correct?
func Skip(msg string, caller int) {
	ginkgo.Skip(msg, caller+1)
}

Reply:
Right

Reply:
Isn't that the same as calling `ginkgo.Skip` directly? Why do we need the wrapper function with a different API?

https://pkg.go.dev/github.com/onsi/ginkgo#Skip